python/utils.py

```diff
@@ -133,20 +130,55 @@ def py_chi2Profile(intensity, Rmin, dR, nxy, dxy, udat, vdat, vis_obs_re, vis_ob
 
     return chi2
 
-def radial_profile(Rmin, delta_R, nrad, mode='Gauss', dtype='float64', gauss_width=100):
+
+def radial_profile(Rmin, delta_R, nrad, mode='Gauss', dtype='float64', gauss_width=1.):
     """ Compute a radial brightness profile. Returns int in Jy/sr """
     gridrad = np.linspace(Rmin, Rmin + delta_R * (nrad - 1), nrad).astype(dtype)
 
     if mode == 'Gauss':
         # a simple Gaussian
-        ints = np.exp(-(gridrad/delta_R/gauss_width)**2)
+        ints = np.exp(-(gridrad/gauss_width)**2)
     elif mode == 'Cos-Gauss':
         # a cos-tapered Gaussian
-        ints = np.cos(2.*np.pi*gridrad/(50.*delta_R))**2. * np.exp(-(gridrad/delta_R/80)**2)
+        ints = np.cos(2.*np.pi*gridrad/(gauss_width))**2. * np.exp(-(gridrad/gauss_width)**2)
 
     return ints
 
 
+def central_pixel(I, Rmin, dR, dxy):
+    """
+    Compute brightness in the central pixel as the average flux in the pixel.
+
+    """
+    # with quadrature method: tends to over-estimate it
+    # area = np.pi*((dxy/2.)**2-Rmin**2)
+    # flux, _ = quadrature(lambda z: f(z)*z, Rmin, dxy/2., tol=1.49e-25, maxiter=200)
+    # flux *= 2.*np.pi
+    # intensmap[int(nrow/2+Dy/dxy), int(ncol/2-Dx/dxy)] = flux/area
+
+    # with trapezoidal rule: it's the same implementation as in galario.cpp
+    iIN = int(np.floor((dxy / 2 - Rmin) // dR))
+    flux = 0.
+    for i in range(1, iIN):
+        flux += (Rmin + dR * i) * I[i]
+
+    flux *= 2.
+    flux += Rmin * I[0] + (Rmin + iIN * dR) * I[iIN]
+    flux *= dR
+
+    # add flux between Rmin+iIN*dR and dxy/2
+    I_interp = (I[iIN + 1] - I[iIN]) / (dR) * (dxy / 2. - (Rmin + dR * (iIN))) + \
+               I[iIN]  # brightness at R=dxy/2
+    flux += ((Rmin + iIN * dR) * I[iIN] + dxy / 2. * I_interp) * (
+                dxy / 2. - (Rmin + iIN * dR))
+
+    # flux *= 2 * np.pi / 2.  # to complete trapezoidal rule (***)
+    area = ((dxy / 2.) ** 2 - Rmin ** 2)
```

Comment:
you can omit the outer parentheses. In general, I can't quite follow the math here. I guess some latex expressions would help. Since it will be hidden in the C++ code, here would be the best place. But would that show up in the docs?

Reply:
> I will write a dedicated docs page

That will surely work. good idea